extensions/amp-story/1.0/page-advancement.js

```diff
@@ -42,6 +42,13 @@ const NEXT_SCREEN_AREA_RATIO = 0.75;
 /** @private @const {number} */
 const PREVIOUS_SCREEN_AREA_RATIO = 0.25;
 
+/**
+ * Protected edges of the screen in pixels.
```

Comment:
Plz explain what "protected" means for someone with no context :)

Reply:
Done.